docs/console.html

```diff
@@ -0,0 +1,804 @@
+                </figcaption>
+                <pre class="prettyprint"><code class="language-javascript">
+                    console.log("%s has %d points", "Sam", "100");
+</code></pre>
+                This would result in <samp>Sam has 100 points</samp> being logged to the console.
+            </figure>
+
+            <p>The full list of format specifiers are as follows:</p>
+
+            <dl>
+                <dt><code>%s</code></dt>
+                <dd>Formats the value as a string</dd>
+
+                <dt><code>%d</code></dt>
+                <dt><code>%i</code></dt>
+                <dd>Formats the value as an integer.</dd>
+
+                <dt><code>%f</code></dt>
+                <dd>Formats the value as a floating point value.</dd>
+
+                <dt><code>%o</code></dt>
+                <dd>Formats the value as an expandable DOM element. As seen in the Elements panel.</dd>
+
+                <dt><code>%O</code></dt>
+                <dd>Formats the value as an expandable JavaScript object.</dd>
+
+                <dt><code>%c</code></dt>
+                <dd>Applies CSS style rules to the output string as specified by the second parameter.</dd>
+            </dl>
+
+            <figure>
+                <figcaption>
+                    This example uses the digit specifier to format the value of <code>document.childNodes.length</code>.
+                    It also uses the floating point specifier to format the value of <code>Date.now()</code>.
+                </figcaption>
+                <figure>
+                    <figcaption>The code:</figcaption>
+                    <pre class="prettyprint"><code class="language-javascript">
+console.log("Node count: %d, and the time is %f.", document.childNodes.length, Date.now());
+</code></pre>
+                </figure>
+                <figure>
+                    <img class="console-files/format-substitution.png" alt="Example subsitution output" />
+                    <figcaption>
+                        The output of the previous code sample.
+                    </figcaption>
+                </figure>
+            </figure>
+        </section>
+
+        <section>
+            <h3 id="formatting-dom-elements-as-javascript-objects">Formatting DOM elements as JavaScript objects</h3>
+
+            <p>
+                When you log a DOM element to the console it is displayed in an XML format.
+                This is the same way this are displayed in the Elements panel.
+                To display the JavaScript representation you may either use the <code>dir()</code> method or the object format specifier in a <code>log()</code> statement.
+            </p>
+
+            <figure>
+                <figcaption>The different views for logging an element.</figcaption>
+                <figure>
+                    <img src="console-files/log-element.png" alt="Element logged" />
```

Comment:
these two screenshots are really big. can we make them more dense and do them side by side?